drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -113,6 +103,9 @@ struct qspi_nor_data {
 
 static inline int qspi_freq_to_sckfreq(int freq)
 {
+	if (freq > (NRF_QSPI_BASE_CLOCK_FREQ)) {
+		return NRF_QSPI_FREQ_DIV1;
```

Comment:
This function input is given in MHz unit. Yet the macro NRF_QSPI_BASE_CLOCK_FREQ is specified in Hz.